packages/javascript-sdk/src/token-manager/helpers.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * @forgerock/javascript-sdk
+ *
+ * helpers.test.ts
+ *
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+import { tokensWillExpireWithinThreshold } from './helpers';
+import {
+  oauthThreshold,
+  tokenExpiryWithinThreshold,
+  tokenExpiryOutsideThreshold,
+} from './token-manager.mock.data';
+
+describe('Test Token Manager utils', () => {
+  it('Should return true if tokens expire within configured threshold', async () => {
```

Comment:
I don't think we need an async callback here.

Reply:
Good point!